src/core/plugins/auth/actions.js

```diff
@@ -139,13 +140,17 @@ export const authorizeAccessCodeWithBasicAuthentication = ( { auth, redirectUrl
   return authActions.authorizeRequest({body: buildFormData(form), name, url: schema.get("tokenUrl"), auth, headers})
 }
 
-export const authorizeRequest = ( data ) => ( { fn, getConfigs, authActions, errActions, authSelectors } ) => {
+export const authorizeRequest = ( data ) => ( { fn, getConfigs, authActions, errActions, oas3Selectors, specSelectors } ) => {
   let { body, query={}, headers={}, name, url, auth } = data
-  let { additionalQueryStringParams } = authSelectors.getConfigs() || {}
```

Comment:
Not sure why remove the additionalQueryStringParams...
Isn't that a desired functionality?

Reply:
`additionalQueryStringParams` is used on line 148 to change the `url` variable, but the `url` variable is not used afterwards. I suggested cleaning it up.

Reply:
`additionalQueryStringParams`, as @luka-zitnik pointed out, is already broken, so I'm going to leave this be for now and continue working on that in https://github.com/swagger-api/swagger-ui/issues/4399.